src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/adsbase/AdsRecoveryObject.java

```diff
@@ -41,7 +41,7 @@ public void assertIfAllRecoveredSlotHasCorrectSizeAndBackground(List<WebElement>
     for (WebElement ad : recoveredAds) {
       Dimension adSize = ad.getSize();
 
-      if (adSize.equals(TOP_LEADERBOARD_SIZE)) {
+      if (adSize.equals(LEADERBOARD_SIZE)) {
         Assertion.assertTrue(ad.getCssValue("background").contains(expectedRecoveredLB), "TOP_LEADERBOARD is not correctly recovered!");
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Define a constant instead of duplicating this literal "background" 4 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AS1192)